pkg/operator/ceph/file/filesystem.go

```diff
@@ -33,9 +33,9 @@ import (
 )
 
 const (
-	dataPoolSuffix     = "data"
-	metadataPoolSuffix = "metadata"
 	appName            = "cephfs"
+	metaDataPoolSuffix = "-metadata"
```

Comment:
Why did you add the `-`? The `fmt.Sprintf()` call already adds the dash. This would cause a double dash

Reply:
removed, yes I observed that it was having double dashes.
Thanks.